### PR TITLE
Add command to export pub/priv key pairs to file

### PR DESCRIPTION
**Pull request by jstnryan**
Giving the user the ability to export their private keys allows wallet portability between the reference client/wallet and other "third-party" wallets.

This feature is currently quite minimal, and doesn't display any warnings in favor of assuming users know the implications of creating copies of private keys. Further enhancements could include such warnings, or the ability to choose which of the available public keys to export.

(Corrects error in earlier pull request #46 )

### CODE REVIEW

File: wallet/main.go [hunk 1]
```diff
@@ -455,6 +456,37 @@ func main() {
...
+				fmt.Printf("Error: %s\n", err)
+				break
+			}
+			count := 0
+			for _, pubKey := range pubKeys {
+				private, err := wallet.GetPrivateKey(pubKey)
+				if err != nil {
+					break
+				}
+				pair := fmt.Sprintf("%s\n%s\n\n",
```
Comment (asdvxgxasjab) [inline wallet/main.go]: I'd maybe separate these with a comma and then a single new line. It'll be easier to use the file with standard tools that scan for single new lines like [`bufio.Scanner`](https://golang.org/pkg/bufio/#Scanner)

File: wallet/main.go [hunk 2]
```diff
@@ -455,6 +456,37 @@ func main() {
...
+			f, err := os.Create(name)
+			if err != nil {
+				fmt.Printf("Error: %s\n", err)
+				break
+			}
+			count := 0
+			for _, pubKey := range pubKeys {
+				private, err := wallet.GetPrivateKey(pubKey)
+				if err != nil {
+					break
```
Comment (asdvxgxasjab) [inline wallet/main.go]: I'd log the error with the public key and then `continue`.

**Review by asdvxgxasjab — COMMENTED** [review]
I think we should probably have an explicit warning. The bitcoin Qt wallet might have a good example for prior art.

File: wallet/main.go (hunk 1 above)
Comment (jstnryan) [reply wallet/main.go]: Changed to "%s,%s\n".

I wonder if it makes sense to define a structure for this and then output as JSON; 
```
type Wallet struct {
    Address ed25519.PublicKey
    Private ed25519.PrivateKey
}
```
It's slightly less human-readable, but perhaps easier to ingest across different platforms.

Comment (asdvxgxasjab) [reply wallet/main.go]: I think the way it's written now is easiest. Don't even need a JSON parser!

**Pull request merged**